manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -49,3 +53,8 @@ spec:
         operator: "Exists"
         effect: "NoExecute"
         tolerationSeconds: 120
+      volumes:
+      - name: images
+        configMap:
+          defaultMode: 420
```

Comment:
is this the proper mode?

`chmod 420` produces `-r---w----` which doesn't look right to me.